backend/passenger.py

```diff
@@ -17,6 +17,38 @@ def __init__(self, valid_positions, random):
 
         self.shortest_path_length = None
 
+class PassengerHotspotLoc:
+    def __init__(self, valid_positions, x, y, random):
+        (x, y) = (x, y)
+        (x_dest, y_dest) = get_valid_passenger_position(valid_positions, random)
+        self.x = x
+        self.y = y
+        self.x_dest = x_dest
+        self.y_dest = y_dest
+        self.car_id = None
+        self.id = str(uuid.uuid4())
+        self.is_in_car = False
+        self.waited_for_car = 0  # time spent outside of a car
+        self.traveled = 0  # time spent in a car
+
+        self.shortest_path_length = None
+
+class PassengerHotspotDest:
+    def __init__(self, valid_positions, x_dest, y_dest, random):
+        (x, y) = get_valid_passenger_position(valid_positions, random)
+        (x_dest, y_dest) = (x_dest, y_dest)
+        self.x = x
+        self.y = y
+        self.x_dest = x_dest
+        self.y_dest = y_dest
+        self.car_id = None
+        self.id = str(uuid.uuid4())
+        self.is_in_car = False
+        self.waited_for_car = 0  # time spent outside of a car
+        self.traveled = 0  # time spent in a car
+
+        self.shortest_path_length = None
+
```

Comment:
These two classes are not needed. We can have one class Passenger with default values or checking in the construction if the parameters were passed

Reply:
only one class for the passengers - parameters are checked on the constructor